plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java

```diff
@@ -29,8 +29,7 @@
 import org.pentaho.di.core.logging.LogChannelInterface;
```

Comment:
Copyright date on this file needs to be updated.

Reply:
fixed.

Reply:
fixed.